docs/src/gallery/tutorials/run_doc2vec_lee.py

```diff
@@ -215,9 +215,15 @@ def read_corpus(fname, tokens_only=False):
 
 ###############################################################################
 # Next, train the model on the corpus.
-# If optimized Gensim (with BLAS library) is being used, this should take no more than 3 seconds.
-# If the BLAS library is not being used, this should take no more than 2
-# minutes, so use optimized Gensim with BLAS if you value your time.
+# In the usual case, where Gensim installation found a BLAS library for optimized
+# bulk vector operations, this training on this tiny 300 document, ~60k word corpus 
+# should take just a few seconds. (More realistic datasets of tens-of-millions
+# of words or more take proportionately longer.) If for some reason a BLAS library 
+# isn't available, training uses a fallback approach that takes 60x-120x longer, 
+# so even this tiny training will take minutes rather than seconds. (And, in that 
+# case, you should also notice a warning in the logging letting you know there's 
+# something worth fixing.) So, be sure your installation uses the BLAS-optimized 
+# Gensim if you value your time.
```

Comment:
A concrete set of instructions for how to actually perform such check would be helpful. E.g. through the `FAST_VERSION` attribute.

I'd also personally drop the brackets, but I realize that's your writing style :)